File: TS/ToolListChangedNotification.ts
/**
 * An optional notification from the server to the client, informing it that the list of tools it offers has changed. This may be issued by servers without any previous subscription from the client.
 */
import { Notification } from './Notification';

export interface ToolListChangedNotification extends Notification {
  method: "notifications/tools/list_changed";
}

File: Sources/Schema/ToolListChangedNotification.swift
/**
 * An optional notification from the server to the client, informing it that the list of tools it offers has changed. This may be issued by servers without any previous subscription from the client.
 */
public struct ToolListChangedNotification: Notification, Codable {
    public var method: String = "notifications/tools/list_changed"
    public var params: NotificationParams?
    
    public init() {
        self.params = nil
    }
}
